powersimdata/data_access/data_access.py

```diff
@@ -115,6 +116,9 @@ def push(self, file_name, checksum):
         """
         raise NotImplementedError
 
+    def get_profile_version(self, grid_model, kind):
+        return ProfileHelper.get_profile_version_cloud(grid_model, kind)
```

Comment:
We implicitly return the versions stored on the cloud. I see how it is useful but is that intuitive in the `DataAccess` class? Should we return instead an instance of `ProfileHelper` and call the functions in the child classes?

Reply:
I was thinking since blob storage is the source of truth, it makes sense as the default. Another way would be to `raise NotImplemented` here and have the child classes contain their own implementations (`SSHDataAccess` is just inheriting this, while `LocalDataAccess` appends the local versions if there are any). Would that be more intuitive?

Reply:
You are right. It makes sense it is the default.